core/model.py

```diff
@@ -124,12 +124,22 @@ def _predict(self, args):
         return self.models[model].predict(x)
 
     def _post_process(self, predictions):
-        if predictions.shape[1] == 12:  # multivariate model
+        if predictions.shape[1] == 15:  # multivariate model
             predictions = np.array(predictions)
-            predictions[:, 0] = np.clip(predictions[:, 0], 0, 10)  # HOURLYVISIBILITY - should be 0-10
+            predictions[:, 0] = np.clip(np.rint(predictions[:, 0]), 0, 10)  # HOURLYVISIBILITY - should be int 0-10
+            predictions[:, 1] = np.clip(predictions[:, 1], -15, 106) # HOURLYDRYBULBTEMPF - should be between extremes recorded in nyc
+            predictions[:, 2] = np.clip(predictions[:, 2], -13, 104) # HOURLYWETBULBTEMPF - should be between extremes recorded in nyc
+            predictions[:, 3] = np.clip(predictions[:, 3], -30, 95) # HOURLYDewPointTempF - should be between extremes recorded in nyc
+            predictions[:, 4] = np.clip(predictions[:, 4], 0, 101) # HOURLYRelativeHumidity - should be between extremes recorded in nyc
```

Comment:
flake8
```suggestion
            predictions[:, 1] = np.clip(predictions[:, 1], -15, 106)  # HOURLYDRYBULBTEMPF - should be between extremes recorded in nyc
            predictions[:, 2] = np.clip(predictions[:, 2], -13, 104)  # HOURLYWETBULBTEMPF - should be between extremes recorded in nyc
            predictions[:, 3] = np.clip(predictions[:, 3], -30, 95)  # HOURLYDewPointTempF - should be between extremes recorded in nyc
            predictions[:, 4] = np.clip(predictions[:, 4], 0, 101)  # HOURLYRelativeHumidity - should be between extremes recorded in nyc
```

Reply:
thanks, I see it now in the travis job where it says `at least two spaces before inline comment`, didn't know about this rule